app/assets/stylesheets/chores.scss

```diff
@@ -1,3 +1,20 @@
 // Place all the styles related to the Chores controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+// Color scheme from https://www.behance.net/gallery/32154055/Minimalist-Color-Palettes-2015
+$a-yellow: #e5eec1;
```

Comment:
Do you have a recommendation for accomplishing the following: the five colors in pallette also have an order to them. I was hoping to embed the order in their names. Should I just leave that out of the names? Or make it more explicit with a name like `$first-yellow` or something similar?